puppet/zulip/manifests/apache_sso.pp

```diff
@@ -1,7 +1,7 @@
 class zulip::apache_sso {
   include zulip::localhost_sso
 
-  case $::osfamily {
+  case $::os['family'] {
     'debian': {
```

Comment:
This apparently relies on the case insensitivity of Puppet string matching (both before and after). I guess that’s fine.